assets/build/install.sh

```diff
@@ -88,12 +88,11 @@ chown -R ${GITLAB_USER}: ${GITLAB_SHELL_INSTALL_DIR}
 
 cd ${GITLAB_SHELL_INSTALL_DIR}
 exec_as_git cp -a config.yml.example config.yml
-if [[ -x ./bin/compile ]]; then
-  echo "Compiling gitlab-shell golang executables..."
-  ./bin/compile
-  rm -rf go_build
-fi
-./bin/install
+
+echo "Compiling gitlab-shell golang executables..."
+sed -i '1s|^|export PATH := '"${GOROOT}"'\/bin:$(PATH)\n\n|' Makefile
+exec_as_git make setup
+rm -rf go_build
```

Comment:
Hey,
thanks for providing the fix for your compling the gitlab shell. 
I did some tests and I think i found a simpler solution.

```bash
echo "Compiling gitlab-shell golang executables..."
exec_as_git bundle install -j"$(nproc)" --deployment --with development test
exec_as_git "PATH=$PATH" make verify setup
```

WDYT ?

Reply:
Hi,

I guess the line

```
exec_as_git bundle install -j"$(nproc)" --deployment --with development test
```

can be ommited, since the installation of the `gitlab-shell` does not depend on ruby, isn't it.

I'm also not really shure if the `make` command should invoke the targets `verify` and `setup`. According to the [notes in the original repo](https://gitlab.com/gitlab-org/gitlab-shell#setup) just the target setup would be enough. 

Did you build the docker image (from scratch) and did you tested the functionality? (I'm already on gitlab 12.6.2.)

Cheers

Reply:
Hi,

I guess your proposed fix should be used, since it invokes some further test that might be helpful. I was wondering about the call of `bundle` but reading the `Makefile` again I figured out that the target `verify` does depend on ruby (and some additional gems) whereas the target `setup` works without installing additional gems - at least up to now.

However, I guess your fix (all lines) should be used!

Thanks and kind regards

Reply:
@sachilles if you want to otherwise I'll do this in the evening.